tests/dialects/test_transform.py

```diff
@@ -78,7 +78,7 @@ def test_tileop_init():
             dynamic_sizes=[],
             static_sizes=static_sizes,
         ),
-        """%0, %1, %2 = "transform.structured.tile_using_for"(%3) <{static_sizes = array<i32: 8, 8>}> : (!transform.any_value) -> (!transform.any_op, !transform.any_op, !transform.any_op)""",
+        """%0, %1, %2 = "transform.structured.tile_using_for"(%3) <{static_sizes = array<i32: 8, 8>}> : (!transform.any_value) -> (!transform.any_op, !transform.any_op, !transform.any_op)""",  # noqa: E501
         None,
     )
```

Comment:
Do we need these noqa comments here since we ignore all test file above (`"tests/*" = ["E501"]`)?
When I remove it, `ruff check` doesn't trigger for me unless I also remove the aforementioned line from `pyproject.toml`.

Reply:
Ah you're right I'll do a pass